src/maven-repair/src/main/java/com/github/tdurieux/repair/maven/plugin/NopolMojo.java

```
@@ -55,6 +56,16 @@
 
 	@Override
     public void execute() throws MojoExecutionException, MojoFailureException {
+        try {
+            // Check if tools.jar is in classpath, as Nopol depends on it.
+            // This is not done in compile time as we don't want Repairnator to it to compile on OpenJDK or JDK >= 9
+            URLClassLoader loader;
+            loader = (URLClassLoader) ClassLoader.getSystemClassLoader();
+            loader.loadClass("com.sun.jdi.AbsentInformationException");
+        } catch (ClassNotFoundException e) {
+            getLog().error("tools.jar has not been loaded, therefore Nopol can't run");
```

throw new RuntimeException(e) instead of logging